components/molecule/buttonGroup/src/index.scss

```diff
@@ -1,4 +1,6 @@
 .sui-MoleculeButtonGroup {
+  display: flex;
+
   &--fullWidth {
     display: flex;
```

Comment:
I guess that if it's always display: flex, then maybe the property for the fullWidth is not needed anymore.

Reply:
this is needed for its children with the class `&--fullWidth` (`AtomButton`'s mostly)